jenkins-slaves/jenkins-slave-zap/.openshift/templates/s2i-dockerfile-build.yml

```diff
@@ -0,0 +1,73 @@
+---
+kind: Template
+apiVersion: v1
+metadata:
+  name: "${NAME}"
+  annotations:
+    openshift.io/display-name: Generic Dockerfile Build
+objects:
+- kind: ImageStream
+  apiVersion: image.openshift.io/v1
+  metadata:
+    name: ${NAME}
+  spec:
+    lookupPolicy:
+      local: false
+- kind: BuildConfig
+  apiVersion: build.openshift.io/v1
+  metadata:
+    labels:
+      build: "${NAME}"
+      template: docker-build-template-binary-with-push-secret
+    name: "${NAME}"
+  spec:
+    failedBuildsHistoryLimit: 5
+    output:
+      to:
+        kind: ImageStream
+        name: "${NAME}:latest"
+    postCommit: {}
+    resources: {}
+    runPolicy: Serial
+    source:
+      contextDir: ${SOURCE_CONTEXT_DIR}
+      git:
+        ref: ${SOURCE_REPOSITORY_REF}
+        uri: ${SOURCE_REPOSITORY_URL}
+      type: Git
+    strategy:
+      dockerStrategy:
+        dockerfilePath: Dockerfile.rhel7-daemon
+        from:
+          kind: DockerImage
+          name: ${BUILDER_IMAGE_NAME}
+      type: Source
```

Comment:
I believe this should be type: Docker

Reply:
Probably correct, but in practice it doesn't seem to matter when I tested it on S11. I will try it with `type: Docker` and see if there is a differnce.

Reply:
Reviewing this again as I may be utilizing this image soon - is there a reason we are not utilizing the [generic jenkins build template](https://github.com/redhat-cop/containers-quickstarts/blob/master/.openshift/templates/jenkins-slave-generic-template.yml)